export default (function(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z,A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,_,$,aa,ab,ac,ad,ae,af,ag,ah,ai,aj,ak,al,am,an,ao,ap,aq,ar,as,at,au,av,aw,ax,ay,az,aA,aB,aC,aD,aE,aF,aG,aH,aI,aJ,aK,aL,aM,aN,aO,aP,aQ,aR,aS,aT,aU,aV,aW,aX,aY,aZ,a_,a$,ba,bb,bc,bd,be,bf,bg,bh,bi,bj,bk,bl,bm,bn,bo,bp,bq,br,bs,bt,bu,bv,bw,bx,by,bz,bA,bB,bC,bD,bE,bF,bG,bH,bI,bJ,bK,bL,bM,bN,bO,bP,bQ,bR,bS,bT,bU,bV,bW,bX,bY,bZ,b_,b$,ca,cb,cc,cd,ce,cf,cg,ch,ci,cj,ck,cl){return {data:{colabs:[{_path:"\u002Fnotebooks\u002Faf_relax_design",_dir:m,_draft:e,_partial:e,_locale:n,_empty:true,title:"AF Relax design",description:"Efficient and scalable de novo protein design using a relaxed sequence space",excerpt:{type:g,children:[]},url:"https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Fsokrypton\u002FColabDesign\u002Fblob\u002Fmain\u002Faf\u002Fexamples\u002Faf_relax_design.ipynb",git:"sokrypton\u002FColabDesign",added:av,type:[t],image:c,category:u,license:s,tags:[J,"protein design"],used_software:[G,v,B],creator:[{name:"Christopher Josef Frank",twitter:d,github:d,orcid:c},{name:"Ali Khoshouei",twitter:d,github:d,orcid:c},{name:"Yosta de Stigter",twitter:d,github:d,orcid:c},{name:"Dominik Schiewitz",twitter:d,github:d,orcid:c},{name:"Shihao Feng",twitter:d,github:d,orcid:c},{name:"Sergey Ovchinnikov",twitter:d,github:d,orcid:c},{name:"Hendrik Dietz",twitter:d,github:d,orcid:c}],body:{type:g,children:[],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:af_relax_design.md",_source:p,_file:"notebooks\u002Faf_relax_design.md",_extension:k,likes:c,stars:249,n_comments:c},{_path:"\u002Fnotebooks\u002Fcharmm-gui_openmm",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"CHARMM-GUI+OpenMM",description:"Using inputs from CHARMM-GUI solution builder and MD simulations with OpenMM",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aw}]},{type:a,tag:f,props:{},children:[{type:b,value:ax},{type:a,tag:q,props:{href:K,rel:[r]},children:[{type:b,value:K}]},{type:b,value:ay},{type:a,tag:q,props:{href:L,rel:[r]},children:[{type:b,value:L}]},{type:b,value:az}]}]},url:"https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Fpablo-arantes\u002Fmaking-it-rain\u002Fblob\u002Fmain\u002FCHARMM_GUI.ipynb",git:"pablo-arantes\u002Fmaking-it-rain",added:av,type:[t],image:"https:\u002F\u002Fraw.githubusercontent.com\u002Fpablo-arantes\u002Fmaking-it-rain\u002Fmain\u002FGraphAbs.png",category:u,license:s,tags:[J,H,k],used_software:[af,v],creator:[{name:"Pablo R. Arantes",twitter:"pablitoarantes",github:"pablo-arantes",orcid:"0000-0003-1946-2750"},{name:"Marcelo D. Polêto",twitter:aA,github:aA,orcid:"0000-0001-9210-690X"},{name:"Conrado Pedebos",twitter:"ConradoPedebos",github:"cpedebos",orcid:c},{name:"Rodrigo Ligabue-Braun",twitter:"ligabue_braun",github:d,orcid:"0000-0002-2555-9754"}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aw}]},{type:a,tag:f,props:{},children:[{type:b,value:ax},{type:a,tag:q,props:{href:K,rel:[r]},children:[{type:b,value:K}]},{type:b,value:ay},{type:a,tag:q,props:{href:L,rel:[r]},children:[{type:b,value:L}]},{type:b,value:az}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:charmm-gui_openmm.md",_source:p,_file:"notebooks\u002Fcharmm-gui_openmm.md",_extension:k,likes:M,stars:M,n_comments:M},{_path:"\u002Fnotebooks\u002Fmolecular_transformer_for_reaction_prediction",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"Molecular Transformer for Reaction Prediction",description:"A short tutorial on how to train a Molecular Transformer model (https:\u002F\u002Fpubs.acs.org\u002Fdoi\u002Ffull\u002F10.1021\u002Facscentsci.9b00576) on the USPTO_MIT dataset.",excerpt:{type:g,children:[{type:a,tag:x,props:{id:ag},children:[{type:b,value:ah}]},{type:a,tag:C,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:aB}]},{type:a,tag:j,props:{},children:[{type:b,value:aC}]},{type:a,tag:j,props:{},children:[{type:b,value:aD}]}]},{type:a,tag:x,props:{id:ai},children:[{type:b,value:aj}]},{type:a,tag:C,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:aE}]},{type:a,tag:j,props:{},children:[{type:b,value:aF}]},{type:a,tag:j,props:{},children:[{type:b,value:aG}]}]},{type:a,tag:x,props:{id:ak},children:[{type:b,value:al}]},{type:a,tag:C,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:aH}]},{type:a,tag:j,props:{},children:[{type:b,value:aI}]},{type:a,tag:j,props:{},children:[{type:b,value:aJ}]}]}]},url:"https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Fschwallergroup\u002Fdmds_language_models_for_reactions\u002Fblob\u002Fmain\u002Fex_reaction_prediction.ipynb",git:"schwallergroup\u002Fdmds_language_models_for_reactions",added:"2023-02-20",type:[t],image:"https:\u002F\u002Fpubs.acs.org\u002Fcms\u002F10.1021\u002Facscentsci.9b00576\u002Fasset\u002Fimages\u002Fmedium\u002Foc9b00576_0009.gif",category:N,license:s,tags:[aK,am],used_software:[an,"OpenNMT"],creator:[{name:"Philippe Schwaller",twitter:"https:\u002F\u002Ftwitter.com\u002Fschwallergroup",github:"https:\u002F\u002Fgithub.com\u002Fschwallergroup",orcid:"0000-0003-3046-6576"}],body:{type:g,children:[{type:a,tag:x,props:{id:ag},children:[{type:b,value:ah}]},{type:a,tag:C,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:aB}]},{type:a,tag:j,props:{},children:[{type:b,value:aC}]},{type:a,tag:j,props:{},children:[{type:b,value:aD}]}]},{type:a,tag:x,props:{id:ai},children:[{type:b,value:aj}]},{type:a,tag:C,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:aE}]},{type:a,tag:j,props:{},children:[{type:b,value:aF}]},{type:a,tag:j,props:{},children:[{type:b,value:aG}]}]},{type:a,tag:x,props:{id:ak},children:[{type:b,value:al}]},{type:a,tag:C,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:aH}]},{type:a,tag:j,props:{},children:[{type:b,value:aI}]},{type:a,tag:j,props:{},children:[{type:b,value:aJ}]}]}],toc:{title:d,searchDepth:h,depth:h,links:[{id:ag,depth:I,text:ah},{id:ai,depth:I,text:aj},{id:ak,depth:I,text:al}]}},_type:o,_id:"content:notebooks:molecular_transformer_for_reaction_prediction.md",_source:p,_file:"notebooks\u002Fmolecular_transformer_for_reaction_prediction.md",_extension:k,likes:c,stars:aL,n_comments:c},{_path:"\u002Fnotebooks\u002Fmetal3d",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"Metal3D",description:"Zinc site prediction with a single structure as input",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aM}]},{type:a,tag:f,props:{},children:[{type:b,value:aN},{type:a,tag:q,props:{href:O,rel:[r]},children:[{type:b,value:O}]}]}]},url:"https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Flcbc-epfl\u002Fmetal-site-prediction\u002Fblob\u002Fmain\u002FMetal3D\u002FColabMetal.ipynb",git:"lcbc-epfl\u002Fmetal-site-prediction",added:"2023-02-19",type:[t],image:c,category:u,license:s,tags:[J,"ligands","metal"],used_software:[v,ao,aO,aP,G],creator:[{name:"Simon L. Dürr",twitter:ap,github:aq,orcid:ar},{name:"Andrea Levy",twitter:"_andrea_levy",github:d,orcid:"0000-0003-1255-859X"},{name:"Ursula Rothlisberger",twitter:"lcbc-epfl",github:d,orcid:"0000-0002-1704-8591"}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aM}]},{type:a,tag:f,props:{},children:[{type:b,value:aN},{type:a,tag:q,props:{href:O,rel:[r]},children:[{type:b,value:O}]}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:metal3d.md",_source:p,_file:"notebooks\u002Fmetal3d.md",_extension:k,likes:c,stars:16,n_comments:c},{_path:"\u002Fnotebooks\u002Fdifflinker",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"DiffLinker: Equivariant Molecular Linker Design",description:"Equivariant 3D-Conditional Diffusion Model for Molecular Linker Design",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aQ}]}]},url:"https:\u002F\u002Fhuggingface.co\u002Fspaces\u002Figashov\u002FDiffLinker",git:"igashov\u002FDiffLinker",added:aR,type:[P],image:c,category:u,license:s,tags:[aS],used_software:[v,G,an],creator:[{name:"Igashov, Ilia",twitter:aT,github:aT,orcid:c},{name:aU,twitter:d,github:d,orcid:c},{name:"Vignac, Clément",twitter:d,github:d,orcid:c},{name:"Satorras, Victor Garcia",twitter:d,github:d,orcid:c},{name:"Frossard, Pascal",twitter:d,github:d,orcid:c},{name:"Welling, Max",twitter:d,github:d,orcid:c},{name:"Bronstein, Michael",twitter:d,github:d,orcid:c},{name:"Correia, Bruno",twitter:d,github:d,orcid:c}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aQ}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:difflinker.md",_source:p,_file:"notebooks\u002Fdifflinker.md",_extension:k,likes:8,stars:139,n_comments:c},{_path:"\u002Fnotebooks\u002Fsmilesdrawer_gradio",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"Smilesdrawer Gradio",description:"How to use SMILES viewer on Gradio",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aV},{type:a,tag:q,props:{href:aW,rel:[r]},children:[{type:b,value:aX}]}]}]},url:"https:\u002F\u002Fhuggingface.co\u002Fspaces\u002Fsimonduerr\u002Fsmilesdrawer",git:"simonduerr\u002Fsmilesdrawer",added:aR,type:[P],image:c,category:N,license:s,tags:[am],used_software:[v,aP],creator:[{name:aY,twitter:ap,github:aq,orcid:ar}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:aV},{type:a,tag:q,props:{href:aW,rel:[r]},children:[{type:b,value:aX}]}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:smilesdrawer_gradio.md",_source:p,_file:"notebooks\u002Fsmilesdrawer_gradio.md",_extension:k,likes:M,stars:c,n_comments:c},{_path:"\u002Fnotebooks\u002Falphafold2_rave_on_1hzb_cold_shock_protein",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"AlphaFold2-RAVE on 1HZB cold shock proein",description:"AlphaFold2-RAVE is an efficient protocol using the structural outputs from AlphaFold2 as initializations for AI augmented molecular dynamics. These simulations result in Boltzmann ranked ensembles.",excerpt:{type:g,children:[{type:a,tag:y,props:{id:aZ},children:[{type:b,value:a_}]},{type:a,tag:f,props:{},children:[{type:b,value:a$}]},{type:a,tag:f,props:{},children:[{type:b,value:ba},{type:a,tag:bb,props:{},children:[{type:b,value:Q}]},{type:b,value:bc}]},{type:a,tag:bd,props:{},children:[{type:a,tag:be,props:{},children:[{type:a,tag:D,props:{},children:[{type:a,tag:z,props:{align:c},children:[{type:b,value:Q}]},{type:a,tag:z,props:{align:E},children:[{type:b,value:bf}]},{type:a,tag:z,props:{align:c},children:[{type:b,value:bg}]},{type:a,tag:z,props:{align:c},children:[{type:b,value:bh}]}]}]},{type:a,tag:bi,props:{},children:[{type:a,tag:D,props:{},children:[{type:a,tag:l,props:{align:c},children:[{type:b,value:bj}]},{type:a,tag:l,props:{align:E},children:[{type:b,value:bk}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:bl}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:F}]}]},{type:a,tag:D,props:{},children:[{type:a,tag:l,props:{align:c},children:[{type:b,value:bm}]},{type:a,tag:l,props:{align:E},children:[{type:b,value:bn}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:F}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:F}]}]}]}]},{type:a,tag:x,props:{id:as},children:[{type:b,value:at}]},{type:a,tag:f,props:{},children:[{type:b,value:bo},{type:a,tag:R,props:{},children:[{type:b,value:S}]},{type:b,value:bp},{type:a,tag:R,props:{},children:[{type:b,value:S}]},{type:b,value:bq}]}]},url:"https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Ftiwarylab\u002Falphafold2rave\u002Fblob\u002Fmain\u002Ffullrunthrough_CSP.ipynb",git:"tiwarylab\u002Falphafold2rave",added:"2023-02-16",type:[t],image:c,category:u,license:s,tags:[k,aK,B],used_software:[B,af,"plumed"],creator:[{name:"Bodhi P. Vani",twitter:d,github:d,orcid:c},{name:"Akashnathan Aranganathan",twitter:d,github:d,orcid:c},{name:"Dedi Wang",twitter:d,github:d,orcid:c},{name:"Pratyush Tiwary",twitter:br,github:br,orcid:c}],body:{type:g,children:[{type:a,tag:y,props:{id:aZ},children:[{type:b,value:a_}]},{type:a,tag:f,props:{},children:[{type:b,value:a$}]},{type:a,tag:f,props:{},children:[{type:b,value:ba},{type:a,tag:bb,props:{},children:[{type:b,value:Q}]},{type:b,value:bc}]},{type:a,tag:bd,props:{},children:[{type:a,tag:be,props:{},children:[{type:a,tag:D,props:{},children:[{type:a,tag:z,props:{align:c},children:[{type:b,value:Q}]},{type:a,tag:z,props:{align:E},children:[{type:b,value:bf}]},{type:a,tag:z,props:{align:c},children:[{type:b,value:bg}]},{type:a,tag:z,props:{align:c},children:[{type:b,value:bh}]}]}]},{type:a,tag:bi,props:{},children:[{type:a,tag:D,props:{},children:[{type:a,tag:l,props:{align:c},children:[{type:b,value:bj}]},{type:a,tag:l,props:{align:E},children:[{type:b,value:bk}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:bl}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:F}]}]},{type:a,tag:D,props:{},children:[{type:a,tag:l,props:{align:c},children:[{type:b,value:bm}]},{type:a,tag:l,props:{align:E},children:[{type:b,value:bn}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:F}]},{type:a,tag:l,props:{align:c},children:[{type:b,value:F}]}]}]}]},{type:a,tag:x,props:{id:as},children:[{type:b,value:at}]},{type:a,tag:f,props:{},children:[{type:b,value:bo},{type:a,tag:R,props:{},children:[{type:b,value:S}]},{type:b,value:bp},{type:a,tag:R,props:{},children:[{type:b,value:S}]},{type:b,value:bq}]}],toc:{title:d,searchDepth:h,depth:h,links:[{id:as,depth:I,text:at}]}},_type:o,_id:"content:notebooks:alphafold2_rave_on_1hzb_cold_shock_protein.md",_source:p,_file:"notebooks\u002Falphafold2_rave_on_1hzb_cold_shock_protein.md",_extension:k,likes:c,stars:13,n_comments:c},{_path:"\u002Fnotebooks\u002Fdiffdock",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"DiffDock",description:"Dock small ligands using a diffusion model",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bs}]}]},url:"https:\u002F\u002Fhuggingface.co\u002Fspaces\u002Fsimonduerr\u002Fdiffdock",git:"simonduerr\u002Fdiffdock",added:"2023-01-25",type:P,image:c,category:N,license:s,tags:[bt,"diffusion"],used_software:[v,G],creator:[{name:"Corso, Gabriele",twitter:bu,github:bu,orcid:c},{name:aU,twitter:"HannesStaerk",github:"  ",orcid:c},{name:"Jing, Bowen",twitter:w,github:w,orcid:c},{name:"Barzilay, Regina",twitter:w,github:w,orcid:c},{name:"Jaakkola, Tommi",twitter:w,github:w,orcid:c}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bs}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:diffdock.md",_source:p,_file:"notebooks\u002Fdiffdock.md",_extension:k,likes:bv,stars:c,n_comments:c},{_path:"\u002Fnotebooks\u002Fopenmm_pdbfixer",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"OpenMM + PDBFixer",description:"Short protocol to fix a PDB file of villin headpiece and run a molecular dynamics simulation",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bw},{type:a,tag:q,props:{href:bx,rel:[r]},children:[{type:b,value:by}]}]}]},url:"https:\u002F\u002Fcolab.research.google.com\u002Fgist\u002Fjchodera\u002F32c6a2049df2d7b5b95703cee9a38158\u002Ffirst_simulation.ipynb",git:"openmm\u002Fopenmm-cookbook",type:t,image:c,category:u,license:s,added:"2023-01-01",tags:[k],used_software:[af],creator:[{name:"John Chodera",twitter:bz,github:bz,orcid:c}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bw},{type:a,tag:q,props:{href:bx,rel:[r]},children:[{type:b,value:by}]}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:openmm_pdbfixer.md",_source:p,_file:"notebooks\u002Fopenmm_pdbfixer.md",_extension:k,likes:c,stars:12,n_comments:c},{_path:"\u002Fnotebooks\u002Frdkit-etkdg",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"Piperazine Ring Conformation using RDKit",description:"RDKIT ETKDG built conformations",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bA}]}]},git:"sunhwan\u002Fblog",url:["https:\u002F\u002Fhub.gke2.mybinder.org\u002Fuser\u002Fsunhwan-blog-om5b2sa9\u002Fnotebooks\u002F_notebooks\u002F2021-02-24-RDKit-ETKDG-Piperazine.ipynb","https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Fsunhwan\u002Fblog\u002Fblob\u002Fmaster\u002F_notebooks\u002F2021-02-24-RDKit-ETKDG-Piperazine.ipynb"],type:["binder",t],category:N,added:"2022-12-02",tags:[am,bt],used_software:[v,an],creator:[{name:"Sunhwan Jo",twitter:bB,github:bB}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bA}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:rdkit-etkdg.md",_source:p,_file:"notebooks\u002Frdkit-etkdg.md",_extension:k,likes:c,stars:aL,n_comments:c},{_path:"\u002Fnotebooks\u002Frasp",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"RaSP - Deep learning stability predictions",description:"Pipeline version for predicting protein variants thermodynamic stability changes ( ΔΔG ) using a deep learning representation.",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bC},{type:a,tag:T,props:{},children:[{type:b,value:bD}]},{type:b,value:w},{type:a,tag:i,props:{className:[U,V]},children:[{type:a,tag:i,props:{className:[W]},children:[{type:a,tag:i,props:{className:[X],ariaHidden:Y},children:[{type:a,tag:i,props:{className:[Z]},children:[{type:a,tag:i,props:{className:[_],style:$},children:[]},{type:a,tag:i,props:{className:[A]},children:[{type:b,value:aa}]},{type:a,tag:i,props:{className:[A,ab]},children:[{type:b,value:ac}]}]}]}]}]},{type:b,value:bE},{type:a,tag:i,props:{className:[U,V]},children:[{type:a,tag:i,props:{className:[W]},children:[{type:a,tag:i,props:{className:[X],ariaHidden:Y},children:[{type:a,tag:i,props:{className:[Z]},children:[{type:a,tag:i,props:{className:[_],style:$},children:[]},{type:a,tag:i,props:{className:[A]},children:[{type:b,value:aa}]},{type:a,tag:i,props:{className:[A,ab]},children:[{type:b,value:ac}]}]}]}]}]},{type:b,value:bF},{type:a,tag:T,props:{},children:[{type:b,value:bG}]},{type:b,value:w},{type:a,tag:q,props:{href:bH,rel:[r]},children:[{type:b,value:bI}]},{type:b,value:bJ},{type:a,tag:q,props:{href:bK,rel:[r]},children:[{type:b,value:bL}]},{type:b,value:bM}]}]},url:"https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002FKULL-Centre\u002Fpapers\u002Fblob\u002Fmain\u002F2022\u002FML-ddG-Blaabjerg-et-al\u002FRaSPLab.ipynb",git:"KULL-Centre\u002Fpapers",type:[t],image:c,category:u,added:"2022-12-01",license:s,tags:["thermostability",J],used_software:[G],creator:[{name:"Matteo Cagiada",twitter:"CagiadaMatteo",github:c,orcid:c}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bC},{type:a,tag:T,props:{},children:[{type:b,value:bD}]},{type:b,value:w},{type:a,tag:i,props:{className:[U,V]},children:[{type:a,tag:i,props:{className:[W]},children:[{type:a,tag:i,props:{className:[X],ariaHidden:Y},children:[{type:a,tag:i,props:{className:[Z]},children:[{type:a,tag:i,props:{className:[_],style:$},children:[]},{type:a,tag:i,props:{className:[A]},children:[{type:b,value:aa}]},{type:a,tag:i,props:{className:[A,ab]},children:[{type:b,value:ac}]}]}]}]}]},{type:b,value:bE},{type:a,tag:i,props:{className:[U,V]},children:[{type:a,tag:i,props:{className:[W]},children:[{type:a,tag:i,props:{className:[X],ariaHidden:Y},children:[{type:a,tag:i,props:{className:[Z]},children:[{type:a,tag:i,props:{className:[_],style:$},children:[]},{type:a,tag:i,props:{className:[A]},children:[{type:b,value:aa}]},{type:a,tag:i,props:{className:[A,ab]},children:[{type:b,value:ac}]}]}]}]}]},{type:b,value:bF},{type:a,tag:T,props:{},children:[{type:b,value:bG}]},{type:b,value:w},{type:a,tag:q,props:{href:bH,rel:[r]},children:[{type:b,value:bI}]},{type:b,value:bJ},{type:a,tag:q,props:{href:bK,rel:[r]},children:[{type:b,value:bL}]},{type:b,value:bM}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:rasp.md",_source:p,_file:"notebooks\u002Frasp.md",_extension:k,likes:c,stars:49,n_comments:c},{_path:"\u002Fnotebooks\u002Fdmasifcolab",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:au,description:"Predict interaction sites using fingerprints",excerpt:{type:g,children:[{type:a,tag:y,props:{id:bN},children:[{type:b,value:au}]},{type:a,tag:f,props:{},children:[{type:b,value:bO}]},{type:a,tag:bP,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:bQ}]},{type:a,tag:j,props:{},children:[{type:b,value:bR}]}]}]},git:"casperg92\u002FMaSIF_colab",url:"https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Fcasperg92\u002FMaSIF_colab\u002Fblob\u002Fmain\u002FdMaSIF_Colab_V1.ipynb",type:t,category:u,added:"2022-07-04",tags:[H,aS,"surfaces"],used_software:[v,ao],creator:[{name:"Casper Goverde",twitter:"CasperGoverde",github:"casperg92"},{name:"Arne Schneuing",twitter:"rneschneuing"},{name:"Freyr Sverisson",twitter:"FreyrSverrisson"},{name:"Hamed Khakzad",twitter:"hamed_khakzad"},{name:"Bruno Correia",twitter:"befcorreia"}],body:{type:g,children:[{type:a,tag:y,props:{id:bN},children:[{type:b,value:au}]},{type:a,tag:f,props:{},children:[{type:b,value:bO}]},{type:a,tag:bP,props:{},children:[{type:a,tag:j,props:{},children:[{type:b,value:bQ}]},{type:a,tag:j,props:{},children:[{type:b,value:bR}]}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:dmasifcolab.md",_source:p,_file:"notebooks\u002Fdmasifcolab.md",_extension:k,likes:c,stars:bv,n_comments:c},{_path:"\u002Fnotebooks\u002Fproteinmpnn",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"ProteinMPNN",description:"Fast fixed backbone sequence redesign",excerpt:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bS}]}]},git:"simonduerr\u002FProteinMPNN",url:"https:\u002F\u002Fhf.space\u002Fsimonduerr\u002FProteinMPNN",type:P,category:u,added:"2022-06-07",tags:[H,"sequence design"],used_software:["3Dmol.js",ao,B,aO],creator:[{name:aY,orcid:ar,twitter:ap,github:aq}],body:{type:g,children:[{type:a,tag:f,props:{},children:[{type:b,value:bS}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:proteinmpnn.md",_source:p,_file:"notebooks\u002Fproteinmpnn.md",_extension:k,likes:59,stars:c,n_comments:I},{_path:"\u002Fnotebooks\u002Fcolabfold_advanced",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"AlphaFold2 Advanced - ColabFold",description:"Advanced AlphaFold pipeline using MMseqs2",excerpt:{type:g,children:[{type:a,tag:y,props:{id:bT},children:[{type:b,value:bU}]},{type:a,tag:f,props:{},children:[{type:b,value:bV}]},{type:a,tag:f,props:{},children:[{type:b,value:ad}]}]},git:bW,url:bX,license:s,type:t,category:u,tags:[H,bY],image:bZ,added:b_,used_software:[v,b$,B],creator:[{name:ca,twitter:ae,github:ae},{name:cb,twitter:cc,github:cd},{name:ce},{name:cf},{name:cg},{name:ch}],body:{type:g,children:[{type:a,tag:y,props:{id:bT},children:[{type:b,value:bU}]},{type:a,tag:f,props:{},children:[{type:b,value:bV}]},{type:a,tag:f,props:{},children:[{type:b,value:ad}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:colabfold_advanced.md",_source:p,_file:"notebooks\u002Fcolabfold_advanced.md",_extension:k,likes:c,stars:ci,n_comments:c},{_path:"\u002Fnotebooks\u002Fcolabfold",_dir:m,_draft:e,_partial:e,_locale:n,_empty:e,title:"AlphaFold2 - ColabFold",description:"Standard AlphaFold pipeline using MMseqs2",excerpt:{type:g,children:[{type:a,tag:y,props:{id:cj},children:[{type:b,value:ck}]},{type:a,tag:f,props:{},children:[{type:b,value:cl}]},{type:a,tag:f,props:{},children:[{type:b,value:ad}]}]},git:bW,url:bX,license:s,type:t,category:u,tags:[H,bY],image:bZ,added:b_,used_software:[v,b$,B],creator:[{name:ca,twitter:ae,github:ae},{name:cb,twitter:cc,github:cd},{name:ce},{name:cf},{name:cg},{name:ch}],body:{type:g,children:[{type:a,tag:y,props:{id:cj},children:[{type:b,value:ck}]},{type:a,tag:f,props:{},children:[{type:b,value:cl}]},{type:a,tag:f,props:{},children:[{type:b,value:ad}]}],toc:{title:d,searchDepth:h,depth:h,links:[]}},_type:o,_id:"content:notebooks:colabfold.md",_source:p,_file:"notebooks\u002Fcolabfold.md",_extension:k,likes:c,stars:ci,n_comments:c}]},prerenderedAt:1677751437377}}("element","text",null,"",false,"p","root",3,"span","li","md","td","notebooks","en","markdown","content","a","nofollow","mit","colab","biology","py3Dmol"," ","h2","h1","th","mord","alphafold","ul","tr","center","✅","pytorch","Bio",2,"proteins","https:\u002F\u002Fwww.charmm-gui.org\u002F","https:\u002F\u002Fwww.youtube.com\u002Fchannel\u002FUCtHN7aNAjDet_JKWPxSioLQ",0,"chemistry","https:\u002F\u002Fdoi.org\u002F10.1101\u002F2022.08.22.504853","huggingface","light_version","sup","*","strong","math","math-inline","katex","katex-html","true","base","strut","height:0.6833em;","ΔΔ","mathnormal","G","Mirdita M, Schütze K, Moriwaki Y, Heo L, Ovchinnikov S, Steinegger M. ColabFold: Making protein folding accessible to all. Nature Methods, 2022","sokrypton","openmm","setup","Setup","opennmt-py-main-steps","OpenNMT-py main steps","additional-stuff","Additional stuff","chemoinformatics","rdkit","torch","simonduerr","duerrsimon","0000-0002-4304-8106","motivation","Motivation","dMaSIF site","2023-02-27","This is a Jupyter notebook for running Molecular Dynamics (MD) simulations using OpenMM engine and CHARMM force field files. This notebook is a supplementary material of the paper \"Making it rain: Cloud-based molecular simulations for everyone\" and we encourage you to read it before using this pipeline.","In general, MD simulations rely on 1) a set of atomic coordinates of all atoms on a simulation box and 2) a set of force field parameters that describes the interaction energies between atoms.\nIn terms of CHARMM inputs, we wil need:\nA .crd and a .pdb file containing a set of atomic coordinates.\nThe respective .psf file containing the system topology.\nA toppar folder containing force field parameter files.\nA toppar.str file pointing to the force field files that we will use in our simulation.\nIn this notebook, we will simulate PDB 1AKI, a hen egg-white lysozyme. To build our simulation box, we will use CHARMM-GUI (",") solution builder feature. (Important: to use CHARMM-GUI, you must register for a free account). You can download the necessary input files from here;\nFor more information on how to create a solvated simulation box, please refer to CHARMM-GUI YouTube channel (",")","mdpoleto","Data download","Load the data","Tokenization","Building the vocab","Training the Molecular Transformer","Testing","Improvements","Further steps","Publications","machine learning",4,"If using please cite:","Accurate prediction of transition metal ion location via deep learning S.L. Dürr, A. Levy, U. Rothlisberger bioRxiv 2022.08.22.504853; doi: ","moleculekit","gradio","Given a set of disconnected fragments in 3D, DiffLinker places missing atoms in between and designs a molecule incorporating all the initial fragments. Our method can link an arbitrary number of fragments, requires no information on the attachment atoms and linker size, and can be conditioned on the protein pockets.","2023-02-18","Geometric deep learning","igashov","Stärk, Hannes","Short demo on how to use SMILESdrawer with Gradio. If using for academic work please cite:  ","https:\u002F\u002Fdoi.org\u002F10.1021\u002Facs.jcim.7b00425","10.1021\u002Facs.jcim.7b00425","Simon Duerr","alphafold2-rave-v-10","AlphaFold2-RAVE v 1.0","This notebook aims to show how efficient our method is in predicting an effective RC (SPIB) from the amino acid sequence of a protein of interest(CSP), that can then be used in enhanced sampling to obtain Boltzmann ranked conformations.","We provide a ","code-inline"," option in our notebook to get a brief overview and the essence of our protocol.","table","thead","Time","MD simulation","SPIB","tbody","True","1  min","❌","False","~ 3 hrs","This protocol is essentially a methodology that combines two schools of thought: structure prediction, and enhanced sampling to preserve thermodynamics. In this repository, we demonstrate one instance of such a methodology. For structure prediction, we use AlphaFold2, or more specifically ColabFold. We introduce stochasticity to the ColabFold algorithm by decreasing the MSA cluster size and generating structures with multiple random seeds",". For thermodynamics, we perform molecular dynamics simulations with metadynamics. To bias the simulations, we learn collective variables from the now-diverse set of structures using SPIB",".","tiwarylab","Implementation of DiffDock, state-of-the-art method for molecular docking, by Gabriele Corso*, Hannes Stark*, Bowen Jing*, Regina Barzilay and Tommi Jaakkola.","drug discovery","gcorso",20,"Part of the ","https:\u002F\u002Fopenmm.github.io\u002Fopenmm-cookbook\u002Fdev\u002Findex.html","OpenMM cookbook","jchodera","3D conformers of molecules containing piperazine (or cyclohexane) using RDKit tend to get a distorted ring conformation. RDKit's ETKDG (Experimental Torsion angle Knowledge-based Distance Geometry) algorithm can be used to fix ring confromations. This notebook demonstrates its use and possible problems. It shows how to filter out non-desirable conformers or improve the RDKit's conformer generation routine.","sunhwan","Preprint pipeline version for predicting protein variants ","thermodynamic stability changes"," using a deep learning representation. The program, using as input a protein structure (uploaded as PDB) returns stability predictions ("," in kcal\u002Fmol) for each variant at each position of the query protein.\nMore details can be found in: ","Blaabjerg et al.:","https:\u002F\u002Fwww.biorxiv.org\u002Fcontent\u002F10.1101\u002F2022.07.14.500157v1","\"Rapid protein stability prediction using deep learning representations\"",". Source code is available on the project ","https:\u002F\u002Fgithub.com\u002FKULL-Centre\u002Fpapers\u002Ftree\u002Fmain\u002F2022\u002FML-ddG-Blaabjerg-et-al","Github"," page.","dmasif-site","Protein binding is determined by the chemical and geometric features between their surfaces. differentiable Molecular Surface Interaction Fingerprinting (dMaSIF) site is a geometric deep learning framework trained on these surface 'fingerprints' to identify potential protein binding sites. For more details, check out the original papers:","ol","Gainza, P., Sverrisson, F., Monti, F., Rodola, E., Boscaini, D., Bronstein, M. M., & Correia, B. E. (2020). Deciphering interaction fingerprints from protein molecular surfaces using geometric deep learning. Nature Methods, 17(2), 184-192.","Sverrisson, F., Feydy, J., Correia, B. E., & Bronstein, M. M. (2021). Fast end-to-end learning on protein surfaces. In Proceedings of the IEEE\u002FCVF Conference on Computer Vision and Pattern Recognition (pp. 15272-15281).","Quickly obtain new sequences for a given backbone. Allows to constrain different parts of the protein or design homomers.","colabfold-advanced","ColabFold - Advanced","This notebook modifies deepmind's original notebook (before AlphaFold-Multimer existed) to add experimental support for modeling complexes (both homo and hetero-oligomers), option to run MMseqs2 instead of Jackhmmer for MSA generation and advanced functionality.","sokrypton\u002FColabFold","https:\u002F\u002Fcolab.research.google.com\u002Fgithub\u002Fsokrypton\u002FColabFold\u002Fblob\u002Fmain\u002FAlphaFold2.ipynb","structure prediction","https:\u002F\u002Fgithub.com\u002Fsokrypton\u002FColabFold\u002Fraw\u002Fmain\u002F.github\u002FColabFold_Marv_Logo.png","2022-06-04","jax","Sergey Ovichnikov","Milot Mirdita","milot_mirdita","milot-mirdita","Konstantin Schütze","Yoshitaka Moriwaki","Lim Heo","Martin Steinegger",1108,"colabfold-alphafold2-using-mmseqs2","ColabFold: AlphaFold2 using MMseqs2","Easy to use protein structure and complex prediction using AlphaFold2 and Alphafold2-multimer. Sequence alignments\u002Ftemplates are generated through MMseqs2 and HHsearch. For more details, see bottom of the notebook, checkout the ColabFold GitHub and read our manuscript."))